metpy/calc/tests/test_tools.py

```diff
@@ -303,6 +303,28 @@ def test_get_layer_float32():
     assert_array_almost_equal(hgt_layer, true_hgt_layer, 4)
 
 
+def test_get_layer_float32_reversed():
```

Comment:
Couldn't this be handled by parameterizing `test_get_layer_float32` on a boolean flag that specifies whether the input should be flipped? Otherwise this looks like it just completely duplicates the old test.